mp/importer/tests/test_batcher.py

```diff
@@ -42,5 +43,92 @@ def test_End_iterator(self):
         self.assertEqual(result, 'abXcX')
 
 
+class TestParser(TestCase):
+
+    def create_test_parser(self):								# simulates the parser within the main script
+        import argparse
+        parser = argparse.ArgumentParser(description='simulates passing arguments to script')
+        parser.add_argument('--db', dest='to_db', help="destination database")			# the main script adds all general arguments (9 more)	
+        parser.add_argument('--from-db', dest='from_db', help="source database")		
+        return parser
+	
+
+    def test_Add_arguments(self):
+        from .. import batcher
+        parser = self.create_test_parser()
+        batcher.add_arguments(parser)					
+        args = ['./bin/import_from_godengo_batcher.py', '--db', 'test_batcher', '--from-db', 'postgresql:///cityscene', '--batch-size', '10', '--max-batches', '2']
+        options = parser.parse_args(args[1:])							# parsing done in the main function and for all arguments except program name
+        result = []  										
+        result.append(options.batch_size)
+        result.append(options.max_batches)
+        self.assertEqual(result, [10, 2])
```

Comment:
why not just:
self.assertEqual(options.batch_size, 10)

rather than adding it to the result?